portfolio/src/main/webapp/script.js

```diff
@@ -27,3 +27,32 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// Written with help from https://www.w3schools.com/howto/howto_html_include.asp
+function createNavigation() {
+    var total, i, element, file, xhttp;
```

Comment:
Also check the differences among const, let and var. And use a proper one for each variable: https://www.w3schools.com/js/js_let.asp